packages/ripple-tide-grant/index.js

```diff
@@ -0,0 +1,89 @@
+import mime from 'mime-types'
+import { getField, getLinkFromField } from '@dpc-sdp/ripple-tide-api/src/services/utils'
+import components from './component-loader'
+
+// Utils
+const formatAudiences = (audiences) => {
+  const formatAudience = (input) => {
+    const term = typeof input === 'string' ? input : input.name
+    if (term) {
+      switch (term) {
+        case 'Individual':
+          return 'individuals'
+        case 'Business':
+          return 'businesses'
+        default:
+          return term.toLowerCase()
+      }
+    }
+  }
+  if (audiences && audiences.length > 0) {
+    const audienceStr = [...new Set(audiences)].map(a => formatAudience(a)).join(', ')
+    return `${audienceStr.charAt(0).toUpperCase() + audienceStr.slice(1)}`
+  }
+  return ''
+}
+
+const formattedSize = (fileSize) => {
+  if (fileSize != null) {
+    // https://stackoverflow.com/a/18650828
+    if (typeof fileSize === 'string') return fileSize
+    if (fileSize === 0) return '0 Bytes'
+    const k = 1024
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+    const i = Math.floor(Math.log(fileSize) / Math.log(k))
+    return parseFloat((fileSize / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i]
+  }
+  return ''
+}
+
+export default {
+  pageComponent: () => import(/* webpackMode: "eager" */ '@dpc-sdp/ripple-tide-grant/index.vue'),
+  components,
+  mapping: {
+    title: 'title',
+    overview: (src) => ({
+      title: getField(src, 'field_overview_title', null),
+      audience: formatAudiences(getField(src, 'field_audience')),
+      funding: getField(src, 'field_node_funding_level', null),
+      startdate: getField(src, 'field_node_dates.value', ''),
+      enddate: getField(src, 'field_node_dates.end_value', ''),
+      description: getField(src, 'field_description.processed', ''),
+      link: getLinkFromField(src, 'field_call_to_action')
+    }),
+    timeline: (src) => ({
+      title: getField(src, 'field_node_timeline.field_paragraph_title'),
+      list: getField(src, 'field_node_timeline.field_timeline').map(timeline => ({
+        title: getField(timeline, 'field_paragraph_title'),
+        subtitle: getField(timeline, 'field_paragraph_cta_text'),
+        url: timeline.field_paragraph_link ? timeline.field_paragraph_link.origin_url || timeline.field_paragraph_link.uri : null,
```

Comment:
Might be worth adding the extra fields from this example to `getLinkFromField` - I wasn't sure if this was specific to the timeline data structure

Reply:
Tried that and it didn't work :/